.travis.yml

```diff
@@ -120,42 +120,44 @@ jobs:
           update: false
           packages:
             - boost
+            - tree
       script:
         - mkdir $BASEPATH/build
         - cd $BASEPATH/build
-        - cmake -DCMAKE_INSTALL_PREFIX=$BASEPATH/../install $BASEPATH
-        - cmake --build . -- -j 2
-        # Do not run tests for osx since it will fail. The reason is that
-        # the precompiled ROOT libraries do not have RPATH set properly.
-        - make install
+        - cmake -DCMAKE_INSTALL_PREFIX=$BASEPATH/_install $BASEPATH
+        - cmake --build . -- -j2
+        - echo "Skipping ctest on MacOS... The reason is that the precompiled ROOT libraries do not have RPATH set properly."
+        - cmake --build . --target install
+        - cd $BASEPATH/_install
+        - brew install tree
+        - tree
 
     - name: gcc7, ROOT v6.16
       os: linux
-      dist: xenial
       compiler: gcc
       env:
-        - CC=gcc-7
-        - CXX=g++-7
         - ROOTBIN="v6.16.00.Linux-ubuntu16-x86_64-gcc5.4"
 
     - name: gcc5, ROOT v6.16 + code coverage
       os: linux
-      dist: xenial
```

Comment:
Removing xenial will change the compiler to gcc7. I think having one legacy build with gcc5 wont hurt.

Reply:
Hmm yes you're right: https://travis-ci.com/github/ComPWA/ComPWA/jobs/352188890#L3841

Reply:
a09cde59b16196ca95e1182982c2c651d320d44f